storage/range_test.go

```diff
@@ -750,6 +750,58 @@ func TestRangeGossipConfigUpdates(t *testing.T) {
 	}
 }
 
+// TestRangeGossipConfigUpdatesInTxn verifies that a write
+// in an uncommitted transaction does not trigger gossip.
```

Comment:
`TestRangeGossipConfigUpdates` is a test for the positive case.

Sure! I've added a test case for `Get`.